test_db.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+import uuid
+from PIL import Image
+import io
+import base64
+import db.queries.user as user
+import db.queries.seed as seed
+import db.queries.picture as picture
+import metadata.picture_set as picture_set_data
+import metadata.picture as picture_data
+import db.__init__ as db
+
+"""
+This is a test script for the database packages. 
+It tests the functions in the user, seed and picture modules.
+"""
+
```

Comment:
Good test overall! If you want to test raising exception when Error handling, you can use assertRaises method.

```python
with self.assertRaises(ValueError) as context:
    #...
```

Reply:
```suggestion
    def test_get_user_id_not_registered(self):
        """
        This test checks if the get_user_id function raises an exception when the user is not registered
        """
        with self.assertRaises(user.UserNotFoundError):
            user.get_user_id(self.cursor, self.email)
```